packages/ui/src/components/Card/index.tsx

```diff
@@ -0,0 +1,74 @@
+import styled from '@emotion/styled'
+import type { ReactNode } from 'react'
+import { Stack } from '../Stack'
+import { Text } from '../Text'
+
+type CardProps = {
+  /**
+   * Children can be a component, but it can also be a function to get disabled and
+   * isActive state from Card.
+   */
+  children:
+    | ReactNode
+    | ((renderProps: { disabled?: boolean; isActive?: boolean }) => ReactNode)
+  /**
+   * Header can be a string but also a component if you need more complex header.
+   */
+  header?: ReactNode
+  /**
+   * isActive enable a primary style on Card component for when you need to highlight it.
+   */
+  isActive?: boolean
+  disabled?: boolean
+  className?: string
+  'data-testid'?: string
+}
+
+const StyledStack = styled(Stack)`
+  &[data-disabled='true'] {
+    cursor: not-allowed;
+  }
+`
+
+const BorderedBox = styled.div`
+  border: 1px solid ${({ theme }) => theme.colors.neutral.border};
+  border-radius: ${({ theme }) => theme.radii.default};
+  padding: ${({ theme }) => theme.space['3']};
+
+  &[data-isActive='true'] {
+    border: 1px solid ${({ theme }) => theme.colors.primary.border};
+  }
+
+  &[data-disabled='true'] {
+    border: 1px solid ${({ theme }) => theme.colors.neutral.borderDisabled};
+  }
+`
+
+export const Card = ({
```

Comment:
How to replace Container with subtitle/rightTitle ? :)

Reply:
@chambo-e You could pass directly into `header` props:

```tsx
<Card header={<Stack gap={1}>
      <Stack gap={1} direction="row" alignItems="center">
        <Text variant="heading" as="h2">
          Advanced Header
        </Text>
        <Badge variant="success" size="small">
          New
        </Badge>
      </Stack>
      <Alert variant="info">This is an info alert in the header</Alert>
    </Stack>}
>
    This is the content of a Card
</Card>
```